src/docs/reference-manual/transforms.Rmd

```diff
@@ -283,19 +283,19 @@ so there is only one call to $\exp(-y)$.
 ## Affinely Transformed Scalar
 
 Stan uses an affine transform to be able to specify parameters with a given
-location and scale.
+offset and multiplier.
```

Comment:
same as above - sorry that `docs` are still hanging around.
these changes need to be made in the `docs` repo.

Reply:
done.